### PR TITLE
Fix bad code typo :/

### PR DESCRIPTION
**Pull request by byroot**
@dylanahsmith @rafaelfranca for review please.

### CODE REVIEW

File: lib/identity_cache/query_api.rb [hunk 1]
```diff
@@ -263,7 +263,7 @@ def prefetch_associations(associations, records)
               raise ArgumentError.new("Embedded belongs_to associations do not support prefetching yet.")
             else
               reflection = details[:association_reflection]
-              if reflection.options[:polymorphic]
+              if reflection.polymorphic?
```
Comment (byroot) [inline lib/identity_cache/query_api.rb]: This one wasn't typoed, but it's just for consistency.

**Comment by byroot** [comment]
I also pushed https://github.com/Shopify/identity_cache/commit/74a72cb2d00c24a70c4c0de5b34059b45761a1ef

This one was a logic error.

File: lib/identity_cache/parent_model_expiration.rb [hunk 2]
```diff
@@ -38,9 +38,10 @@ def parents_to_expire_on_changes(parents_to_expire, association_name, cached_ass
       old_parent = nil
       if transaction_changed_attributes[foreign_key].present?
         begin
-          if parent_association.options[:polymorhpic]
```
Comment (dylanahsmith) [inline lib/identity_cache/parent_model_expiration.rb]: oops

**Comment by rafaelfranca** [comment]
:shipit:

**Comment by rafaelfranca** [comment]
:clap:

**Pull request merged**